lib/otr.js

```diff
@@ -583,45 +585,53 @@
       case CONST.MSGSTATE_FINISHED:
         this.storedMgs.push({msg: msg, meta: meta})
         this.notify('Message cannot be sent at this time.', 'warn')
-        return
+        return promise
       case CONST.MSGSTATE_ENCRYPTED:
         msg = this.prepareMsg(msg)
         break
       default:
         throw new Error('Unknown message state.')
```

Comment:
Probably want to reject here.

Reply:
Throwing an exception is the same as rejecting.